pkg/exchange/bitget/stream_test.go

```diff
@@ -106,6 +106,22 @@ func TestStream(t *testing.T) {
 		<-c
 	})
 
+	t.Run("kline test", func(t *testing.T) {
+		s.Subscribe(types.KLineChannel, "BTCUSDT", types.SubscribeOptions{Interval: types.Interval1w})
+		s.SetPublicOnly()
+		err := s.Connect(context.Background())
+		assert.NoError(t, err)
+
+		s.OnKLine(func(kline types.KLine) {
+			t.Log("got update", kline)
+		})
+		s.OnKLineClosed(func(kline types.KLine) {
+			t.Log("got closed update", kline)
+		})
+		c := make(chan struct{})
+		<-c
```

Comment:
this test will be blocked?

Reply:
yeah~but it will be skipped in CI tests. if we want to see something, we can run this test to view the results.